packages/qwik-city/buildtime/markdown/frontmatter.ts

```diff
@@ -2,14 +2,18 @@ import type { Transformer } from 'unified';
 import type { BuildContext, FrontmatterAttrs } from '../types';
 import { normalizePath } from '../utils/fs';
 import { visit } from 'unist-util-visit';
+import { parse as parseYaml } from 'yaml';
 import type { ResolvedDocumentHead } from '../../runtime/src';
 
 export function parseFrontmatter(ctx: BuildContext): Transformer {
   return (mdast, vfile) => {
     const attrs: FrontmatterAttrs = {};
 
     visit(mdast, 'yaml', (node: any) => {
-      parseFrontmatterAttrs(attrs, node.value);
+      const parsedAttrs = parseFrontmatterAttrs(node.value);
+      for (const k in parsedAttrs) {
```

Comment:
I am a bit confused. Getting `parsedAttrs` and then coping it over to `attrs` seems like an unnecessary step. Can we just assign directly to `attrs`?

Reply:
It is indeed unnecessary but that's how it was before and I maintained that. In theory it let you specify multiple frontmatters in the file, so all variables will just append (but who wants that anyway? haha)